tla-import/src/main/scala/at/forsyte/apalache/io/tlc/config/TlcConfig.scala

```diff
@@ -20,13 +25,60 @@ case class InitNextSpec(init: String, next: String) extends BehaviorSpec
   */
 case class TemporalSpec(name: String) extends BehaviorSpec
 
+/**
+  * An unspecified behavior spec.
+  */
+case class NullSpec() extends BehaviorSpec
+
+/**
+  * A constant expression that can be written in the right-hand side of an assignment.
+  */
+abstract sealed class ConfigConstExpr {
+  /**
+    * Convert the expression to the intermediate representation.
+    * @return the TLA IR expression that represents the parsed constant expression
+    */
+  def toTlaEx: TlaEx
+}
+
+/**
+  * A TLC model value, that is, a unique identifier that is treated as an uninterpreted constant.
+  * @param name the name of a model value
+  */
+case class ConfigModelValue(name: String) extends ConfigConstExpr {
+  override def toTlaEx: TlaEx = ValEx(TlaStr(name))
+}
```

Comment:
I see a problem here: from the user perspective, model values and string constants are different entities; However, as implemented, a model value A, and a string value "A" result in the same LIR expression. What about encoding model values with some special symbol, e.g. $?

```suggestion
case class ConfigModelValue(name: String) extends ConfigConstExpr {
  override def toTlaEx: TlaEx = ValEx(TlaStr(name))
}
```
```suggestion
case class ConfigModelValue(name: String) extends ConfigConstExpr {
  override def toTlaEx: TlaEx = ValEx(TlaStr("$" + name))
}
```

Reply:
Good point, I will add a prefix like `ModelValue_`. I see model values more like a hack, as they are semantically just uninterpreted constants, not so different from strings in TLA+.

Reply:
agree; it's a hack:) but still good to differentiate between the two types, such that the user doesn't run into some unexpected errors.

Reply:
Returning back to this: I think adding the `ModelValue_` prefix would be an overkill. Imagine Apalache returning counterexamples like `your_set = {"ModelValue_A", "ModelValue_B", "ModelValue_C"}` 

I would really prefer to have strings and model values differentiated, but in a non-overwhelming way, like with a small prefix: `$`, `#`, `_` ... But even leaving the strings clashing with the model values would be still better than the `ModelValue_` prefix.

Reply:
We are already using `$` when renaming variables. I believe lots of unusual ASCII symbols confuse users a lot. Here it is clear that the string is coming from the config as a model value :)